# 
# Submitted-By: Benedikt Spranger <b.spranger@linutronix.de> 2005-01-31
# Committed-by: Robert Schwebel <r.schwebel@pengutronix.de>
# 
# Error: 
#
# Malloc and free seem not to be supported on Solaris 2.7; however, on
# sane systems they are allready there. 
#
# Description: 
#
# unknown
#
# State:
# 
# unknown
#

--- coreutils-5.0/lib/putenv.c.orig	2005-01-31 19:26:59.694110719 +0100
+++ coreutils-5.0/lib/putenv.c	2005-01-31 19:29:20.290050397 +0100
@@ -31,8 +31,10 @@
    with a non-const argument.  That would conflict with the declaration of
    rpl_putenv below (due to the #define putenv rpl_putenv from config.h).  */
 
+#ifdef PLEASE_DEFINE_IF_ON_SOLARIS_2_7 /* we are NOT on Solaris 2.7 */
 char *malloc ();
 void free ();
+#endif
 
 #if defined (__GNU_LIBRARY__) || defined (HAVE_STRING_H)
 # include <string.h>
